mail/src/test/java/com/sun/mail/util/logging/MailHandlerTest.java

```diff
@@ -618,6 +600,37 @@ public void testPublish() {
 
         instance.close();
     }
+    
+    @Test
+    public void testPublishNull() {
+    	MailHandler instance = new MailHandler();
+    	InternalErrorManager em = new InternalErrorManager();
+    	instance.setErrorManager(em);
+    	instance.setLevel(Level.ALL);
+    	instance.publish((LogRecord) null);
+    	instance.close();
+    	for (Throwable t : em.exceptions) {
+            dump(t);
+        }
+        assertEquals(true, em.exceptions.isEmpty());
```

Comment:
assertEquals(true, ... ought to be just assertTrue(...

Reply:
Yes.  There are multiple places I need to fix this.  I'll fix it in the new test and do a sweep of this in a later change.